src/plugins/data/public/search/search_source/search_source.ts

```diff
@@ -439,7 +439,7 @@ export class SearchSource {
 
       // request the remaining fields from both stored_fields and _source
       const remainingFields = difference(fields, keys(body.script_fields));
-      body.stored_fields = remainingFields;
```

Comment:
don't think we can do this so easily without breaking something, but with this change alias fields work